### DISCUSSION TITLE
Specifically let users specify a alternative API URL for the app.

### DISCUSSION THREAD

**Discussion by JamesClarke7283** [Docile-Alligator/Infinity-For-Reddit#1685] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

I think it would be awesome if we could change the base URL for the reddit API. so people can use unnofficial api's that don't have the fees, it would use the same schema as the official API.

If such projects get made to mimic the Reddit API, then it will be quite useful.
DISCLAIMER: I intend to make one if one does not already exist.

### Alternatives

Ive considered just the ability to set your own API keys, but that was already discussed.

### Additional context

I have seen the feature idea of being able to set the API key, but this goes 1 step further.

### How important is this feature?

Important

**Comment by trlef19** [comment]
He has said that reddit doesn't let him. You can find it if you find the post about new reddit api charges he made a year ago
